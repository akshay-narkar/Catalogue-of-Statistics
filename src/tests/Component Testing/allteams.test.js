import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Allteam from '../../Components/allteams';

const x = {
  id: 2,
  area: { id: 25, name: 'X' },
  crestUrl: 'X',
  name: 'abc',
};

it('UI is unaltered wrt first snapshot', () => {
  const elem = renderer.create(<BrowserRouter><Allteam x={x} /></BrowserRouter>).toJSON();

  expect(elem).toMatchSnapshot();
});
