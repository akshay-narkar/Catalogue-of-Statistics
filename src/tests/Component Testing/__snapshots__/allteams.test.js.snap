// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI is unaltered wrt first snapshot 1`] = `
<a
  className="link"
  href="/team"
  onClick={[Function]}
>
  <div
    className="d-flex align-items-center justify-content-start px-5"
    id={2}
    style={
      Object {
        "width": "310px",
      }
    }
  >
    <img
      alt="LogoHere"
      src="X"
      style={
        Object {
          "width": "85px",
        }
      }
    />
    <div>
      <p
        className="ms-3 fw-bold"
      >
        abc
      </p>
      <p
        className="ms-3 fw-light"
      >
        X
      </p>
    </div>
  </div>
</a>
`;
